src/www/diag_dns.php

```diff
@@ -207,14 +207,14 @@ function display_host_results ($address,$hostname,$dns_speeds) {
 							          <td>
 								      <? if($found > 0): ?>
 											  <?PHP	if($alias_exists): ?>
-												An alias already exists for the hostname <?= htmlspecialchars($host) ?>. <br />
+												<?=gettext("An alias already exists for the hostname"); ?> <?= htmlspecialchars($host) ?>. <br />
```

Comment:
Always wrap the whole sentence for context using sprintf to bring dynamic elements into the translation string:

    <?= sprintf(gettext(-An alias already exists for the hostname %s.'), htmlspecialchars($host)) ?>

(same below)